rest-api-spec/src/main/resources/rest-api-spec/api/rank_eval.json

```diff
@@ -1,26 +1,36 @@
-{ 
-  "rank_eval": { 
-    "documentation": "https://www.elastic.co/guide/en/elasticsearch/reference/master/search-rank-eval.html", 
-    "methods": ["POST"], 
-    "url": { 
-      "path": "/_rank_eval", 
-      "paths": ["/_rank_eval", "/{index}/_rank_eval", "/{index}/{type}/_rank_eval"], 
+{
+  "rank_eval": {
+    "documentation": "https://www.elastic.co/guide/en/elasticsearch/reference/master/search-rank-eval.html",
+    "methods": ["GET", "POST"],
+    "url": {
+      "path": "/_rank_eval",
+      "paths": ["/_rank_eval", "/{index}/_rank_eval"],
       "parts": {
         "index": {
          "type": "list",
          "description" : "A comma-separated list of index names to search; use `_all` or empty string to perform the operation on all indices"
-        }, 
-        "type": {
-          "type" : "list",
-          "description" : "A comma-separated list of document types to search; leave empty to perform the operation on all types"
```

Comment:
I see you are also removing support for type and adding support for GET. I guess the spec was outdated?

Reply:
ok just checking that we only adapted the SPEC to the existing code here, which I think is the case.